src/shared/components/form-components/FormFooter.scss

```diff
@@ -0,0 +1,14 @@
+.hacDev-form-footer {
+  padding: var(--pf-global--spacer--md) (--pf-global--gutter/2);
+  @media (min-width: 768px) {
+    padding: var(--pf-global--spacer--md) (--pf-global--gutter);
+  }
```

Comment:
I don't know if this is the issue, but the sticky footer doesn't look nice at the moment:

![footer](https://user-images.githubusercontent.com/139310/143013023-185784b2-ab38-4b8d-a548-cdacfb6f8153.gif)

I didn't find a quick-fix solution. We can merge this now, but then we should create a follow-up bug to solve this next sprint. Wdyt?

Reply:
I agree, let us merge this for now. I think this is what we have in console. But we can try to make it better with a follow up PR if we want.